docs/react/i18n.en-US.md

```diff
@@ -56,7 +56,7 @@ The following languages are currently supported:
 | Italian                  | it_IT    |
 | Icelandic                | is_IS    |
 | Japanese                 | ja_JP    |
-| Kurdish (Iraq)           | ku_IQ    |
+| Kurdish (Iraq)           | kmr_IQ   |
```

Comment:
Maybe `Kurdish (Kurmanji)` is better here, thus we can add `Kurdish (Sorani)` when we want to add Sorani dialect